### PR TITLE
Move play / pause buttons to the side of animated time-series

### PR DESCRIPTION
**Pull request by matallo**
re: https://github.com/CartoDB/cartodb/issues/12387

---

Cartodb PR: https://github.com/CartoDB/cartodb/pull/12482
Deep-insights.js PR: https://github.com/CartoDB/deep-insights.js/pull/561
Cartodb.js PR: https://github.com/CartoDB/cartodb.js/pull/1701

---

### CODE REVIEW

**Comment by matallo** [comment]
tests pending!

**Comment by xavijam** [comment]
Why didn't we add the tests?

File: src/geo/map/torque-layer.js [hunk 1]
```diff
@@ -30,6 +33,20 @@ var TorqueLayer = LayerModelBase.extend({
     time: undefined // should be a Date instance
   },
 
+  // Helper method to be used from a few places, it parses torque cartocss to get
+  // the animation duration or a default duration
+  getAnimationDuration: function (cartocss) {
```
Comment (xavijam) [inline src/geo/map/torque-layer.js]: Instead of using RegExp, let's keep using a CSS parser (like in turbo-carto, deep-insights).

**Comment by xavijam** [comment]
Tests added @matallo, what's your opinion?

**Comment by matallo** [comment]
🚀

**Pull request merged**